MetaMask/metamask-extension
# feat: Provide chainId prop for Asset List

## **Description**

This PR sets shows the network badge specific to the asset, not the globally selected chain ID.  Of course for now, that chainId is shimmed to be the current chainId until we have the Accounts API providing those assets in correct form

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27724?quickstart=1)

## ** ToDo **
- [x] Ensure that the AssetPicker in Send and Swaps looks correct
- [x] Do the same for the Assets Detail page

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to token list and check the network badge 
2. Go to asset picker and check the network badge
3. Go to asset details and check the network badge

## **Screenshots/Recordings**



### **Before**

<img width="426" alt="SCR-20241009-irdt" src="https://github.com/user-attachments/assets/b37c48e9-abaa-47ab-ad75-387ade440f61">
<img width="344" alt="SCR-20241009-irgl" src="https://github.com/user-attachments/assets/85e818c2-f585-428d-a9fd-2ede37568133">


### **After**

Should be the same
<img width="358" alt="Screenshot 2024-10-17 at 20 37 16" src="https://github.com/user-attachments/assets/6a06703e-f7f0-4072-a80c-e47d726dba7b">

<img width="351" alt="Screenshot 2024-10-17 at 20 37 47" src="https://github.com/user-attachments/assets/6cbdfd4f-aa77-4b1a-8beb-9f5da2b8d12a">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
